doc/md/faq.md

```diff
@@ -532,4 +533,90 @@ func main() {
 	})
 	// ...
 }
+```
+
+
+#### How to store Protobuf objects in a BLOB column?
+
+:::info
+This solution relies on a recent bugfix that is currently available on the `master` branch and
+will be released in `v.0.8.0`
+:::
+
+
+Assuming we have a Protobuf message defined:
+```protobuf
+syntax = "proto3";
+
+package pb;
+
+option go_package = "project/pb";
+
+message Hi {
+  string Greeting = 1;
+}
+```
+
+We add receiver methods to the generated protobuf struct such that it implements [ValueScanner](https://pkg.go.dev/entgo.io/ent/schema/field#ValueScanner)
+
+```go
+func (x *Hi) Value() (driver.Value, error) {
+	return proto.Marshal(x)
+}
+
+func (x *Hi) Scan(src interface{}) error {
+	if src == nil {
+		return nil
+	}
+	if b, ok := src.([]byte); ok {
+		if err := proto.Unmarshal(b, x); err != nil {
+			return err
+		}
+		return nil
```

Comment:
```suggestion
		return proto.Unmarshal(b, x)
```

Reply:
Not sure if it's preferred, because it's less explicit. You can choose to accept it or reject it - both are fine.

Reply:
Interesting suggestion, but  I think I prefer the more explicit version